MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift

```diff
@@ -26,79 +26,125 @@ let b = 32
 */
 // write your code here
 
+print(a >= b)
 
 /*: Question 2
 ### 2. Print the result of a modulo b is equal to zero
 */
 // write your code here
 
-
+print((a % b) == 0)
 /*: Question 3
 ### 3. Print the result of y times b less than or equal to a
 */
 // write your code here
 
-
+print((y * b) <= a)
 /*: Question 4
 ### 4. Print the inverse of a greater than or equal to b
 */
 // write your code here
 
-
+print( a <= b)
 
 /*: Question 5
 ### 5. Print "true" if a modulo b is equal to zero
 */
 // write your code here
 
-
+if (a % b) == 0 {
+    print("true")
+}
 /*: Question 6
 ### 6. Print "true" if a divided by b is greater than x
 */
 // write your code here
 
+if (a / b) > Int(x) {
+    print("true")
+}
 
+// I changed "X" due to it not matching and you can only do match on the type
 
 /*: Question 7
 ### 7. Print "true" if y divided by x is greater than three, otherwise print false
 */
 // write your code here
 
 
+if (Double(y) / x) > 3 {
+    print("true")
+}
 
-
+// I changed "Y" due to it not matching and you can only do match on the type
 /*: Question 8
 ### 8. Print "true" if y is greater than x and a divided by b is greater than 9
 */
 // write your code here
 
-
+if (y > Int(x)) && (a / b) > 9 {
+    print("true")
+}
 
 
 /*: Question 9
 ### 9. Write a function "isGreater" that takes two Int arguments and returns true if the first is greater than the second and false if they're not
 */
 // write your code here
 
+func isGreater(arg1: Int, arg2: Int) -> String{
+    if arg1 > arg2 {
+        return "true"
+    } else {
+        return "false"
+    }
+}
 
-
-
+isGreater(arg1: a, arg2: b)
+isGreater(arg1: y, arg2: a)
 
 
 /*: Question 10
 ### 10. Write a function "isForceWith" that takes a String argument and returns true if the argument is the name of someone with whom the force is strong, and otherwise returns false. People who have the force are Luke, Leia, Anakin, Obi Wan, Yoda, Vader.
 */
 // write your code here
 
-
+func isForceWith(arg : String) -> Bool {
+    switch arg {
+    case "Luke":
+        return true
+    case "Leia":
+        return true
+    case "Anakin":
+        return true
+    case "Obi Wan":
+        return true
+    case "Yoda":
+        return true
+    case "Vader":
+        return true
+    default:
+        return false
+    }
+}
+
+isForceWith(arg: "Steve")
+
+isForceWith(arg: "Anakin")
 
 /*: Question 11
 ### 11. Create a function where the two arguments represent different bank account (one of those bank accounts is yours). What type should these arguments be if we are to then perform some math operations on them? Setup a conditional that will add 10 to your funds and minus 10 from the other funds (bank account) if the other account won't go negative if we were to take 10 dollars from it.
 */
 // write your code here
 
+func bankAcc(arg1: Int, arg2: Int){
```

Comment:
Being nit picky here  but I would name the function variables differently, something like: `myBankAccount` and `otherBankAccount`.